src/components/internal/ContextualModal.tsx

```diff
@@ -0,0 +1,21 @@
+import { FocusScope } from "react-aria";
+import { Css } from "src/Css";
+import { useTestIds } from "src/utils";
+import { ContextualModalProps } from "../ButtonModal";
+
+export function ContextualModal(props: ContextualModalProps) {
+  const { content, title } = props;
+  const tid = useTestIds(props, "popup");
+  return (
+    <FocusScope restoreFocus autoFocus>
+      <div css={Css.p3.df.fdc.myPx(4).gap3.bgWhite.bshModal.br4.maxh("inherit").overflowAuto.$}>
```

Comment:
**question:** What is the `myPx(4)` doing? Also curious about the `maxh("inherit")`.

Reply:
I was looking at the `Menu` component and trying to match certain properties, but I think I can remove the `margin-top/margin-bottom`... 

I added the `maxh("inherit")` from a comment in the `Menu` component that mentioned using it to allow us to take advantage of the height set on the overlay container, so I thought it made sense to use it here.

Reply:
Sounds good.